graph-db/serverless/src/utils/cmr/fetchPageFromCMR.js

```diff
@@ -2,15 +2,21 @@ import fetch from 'node-fetch'
 
 import { indexPageOfCmrResults } from './indexPageOfCmrResults'
 
+let scrollNum = 0
+
 /**
  * Fetch a page of collections from CMR search endpoint and initiate or continue scroll request
  * @param {String} scrollId An optional scroll-id given from the CMR
  * @param {String} token An optional Echo Token
+ * @param {String} providerId CMR provider id whose collections to bootstrap, null means all providers.
```

Comment:
This documentation block is missing the `gremlinConnection` parameter

Reply:
Added.